extensions/wikia/ImageReview/ImageReviewEventsHooks.class.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+use \Wikia\Logger\WikiaLogger;
+use \Wikia\Tasks\Tasks\ImageReviewTask;
+
+class ImageReviewEventsHooks {
+	public static function onUploadComplete( UploadBase $form ) {
+		static::createAddTask( $form->getTitle() );
+
+		return true;
+	}
+
+	public static function onFileRevertComplete( Page $page ) {
+		static::createAddTask( $page->getTitle() );
+
+		return true;
+	}
+
+	public static function onArticleUndelete( Title $title, $created, $comment ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$created, $comment". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)